src/helpers/HelperEvents.js

```diff
@@ -0,0 +1,54 @@
+export default function HelperEvents() {
+  let Carousel_1 = "https://cdn.joinnus.com/files/2022/01/drquhEGIghYW6LQ.jpg"
+  const arrCategories = [
+    { title: "concert", id: 1 },
+    { title: "marathon", id: 2 },
+    { title: "sport", id: 3 },
+  ]
+  let DateEvent = new Date()
+  const arrEvents = [
```

Comment:
Why are these titles being burned directly on the code? Shouldn't the BE take care of sending these data to the FE?

Reply:
Yes, it is only mock to test the helper, when it connects to the back it will change that and it will be handled only by back.

Reply:
sounds good